rust/codelist-rs/src/metadata/metadata.rs

```diff
@@ -145,7 +144,7 @@ mod tests {
         assert!(time_difference < 1000);
         let time_difference = get_time_difference(metadata.provenance.last_modified_date);
         assert!(time_difference < 1000);
-        assert_eq!(metadata.provenance.contributors, HashSet::new());
+        assert_eq!(metadata.provenance.contributors, IndexSet::new());
```

Comment:
i think we should add in a test here were we are checking that when we add and remove contributors that the order is maintained. It does not need to be overly tested (otherwise we are just testing `Indexset` but lets check it is doing what we think. 

something like:

- add contributor A, assert contributor = A
- add contributor B, assert contributor = A, B
- add contributor C, assert A, B, C
- remove B, assert A C in this order